content/en/agent/logs/proxy.md

```diff
@@ -60,17 +69,219 @@ The parameter above can also be set with the following environment variable:
+# Datadog's public endpoints.
+
+backend datadog-logs
+    balance roundrobin
+    mode tcp
+    option tcplog
+    server datadog agent-intake.logs.datadoghq.eu:443 ssl verify required ca-file /etc/ssl/certs/ca-bundle.crt check port 443
+```
+
+**Note**: Download the certificate with the following command:
+
+* `sudo apt-get install ca-certificates` (Debian, Ubuntu)
+* `yum install ca-certificates` (CentOS, Redhat)
+
+The file might be located at `/etc/ssl/certs/ca-bundle.crt` for CentOS, Redhat.
+
+Once the HAProxy configuration is in place, you can reload it or restart HAProxy. **It is recommended to have a `cron` job that reloads HAProxy every 10 minutes** (usually doing something like `service haproxy reload`) to force a refresh of HAProxy's DNS cache, in case `app.datadoghq.eu` fails over to another IP.
 
-[1]: /agent/logs/?tab=tailexistingfiles#send-logs-over-https
-[2]: /agent/proxy
 {{% /tab %}}
 {{< /tabs >}}
 
+#### Agent configuration
+
+Then edit the `datadog.yaml` Agent configuration file and set `logs_no_ssl` to `true`. This is needed as Haproxy does not simply forward the traffic and is not the Datadog backend so cannot use the same certificate.
+
+**Note**: `logs_no_ssl` might set to true because Haproxy is configured to encrypt the data. Do not set this parameter to `true` otherwise.
+
+```
+logs_config:
+  logs_dd_url: "<PROXY_SERVER_DOMAIN>:10514"
+  logs_no_ssl: true
+```
+
+### Using NGINX as a TCP Proxy for logs
+
+This example explains how to configure the Datadog Agent to send logs in TCP to a server with NGINX installed and listening on port `10514` to then forward the logs to Datadog.
+
+`agent ---> nginx ---> Datadog`
+
+#### NGINX configuration
+
+This example `nginx.conf` can be used to proxy Agent traffic to Datadog. The last server block in this configuration does TLS wrapping to ensure internal plaintext logs are encrypted between your proxy and Datadog's log intake API endpoint:
+
+{{< tabs >}}
+{{% tab "Datadog US site" %}}
+
+```conf
+user nginx;
+worker_processes auto;
+error_log /var/log/nginx/error.log;
+pid /run/nginx.pid;
+
+events {
+    worker_connections 1024;
+}
+# TCP Proxy for Datadog Agent
+stream {
+    server {
+        listen 10514; #listen for logs
+        proxy_ssl on;
+        proxy_pass agent-intake.logs.datadoghq.com:10516;
+    }
+}
+```
+
+{{% /tab %}}
+{{% tab "Datadog EU site" %}}
+
+```conf
+user nginx;
+worker_processes auto;
+error_log /var/log/nginx/error.log;
+pid /run/nginx.pid;
+
+events {
+    worker_connections 1024;
+}
+# TCP Proxy for Datadog Agent
+stream {
+    server {
+        listen 10514; #listen for logs
+        proxy_ssl on;
+        proxy_pass agent-intake.logs.datadoghq.eu:443;
+    }
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+#### Agent configuration
+
+Then edit the `datadog.yaml` Agent configuration file and set `logs_config.logs_dd_url` to use the newly created proxy instead of establishing a connection directly with Datadog:
```

Comment:
```suggestion
Edit the `datadog.yaml` Agent configuration file and set `logs_config.logs_dd_url` to use the newly created proxy instead of establishing a connection directly with Datadog:
```